src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.dao;
```

Comment:
in the mate/academy/Main.main() method create instances of new services using injector and test all their methods